jwst/source_catalog/source_catalog.py

```diff
@@ -966,7 +964,20 @@ def catalog(self):
         catalog = self.format_columns(catalog)
 
         # update metadata
-        self.meta['version']['jwst'] = jwst_version
+        # photutils 1.11.0 will change the meta dict key name from
+        # version to versions; can remove this check when we require
+        # photutils >= 1.11.0
+        verkeys = ('versions', 'version')
+        verkey = set(self.meta.keys()).intersection(verkeys).pop()
+        verdict = self.meta.get(verkey, None)
+
+        if verdict is not None:
+            verdict['jwst'] = jwst_version
+            packages = ['Python', 'numpy', 'scipy', 'astropy', 'photutils',
+                        'gwcs', 'jwst']
+            verdict = {key: verdict[key] for key in packages if key in verdict}
+            self.meta[verkey] = verdict
+
```

Comment:
```suggestion
        self.meta['calibration_software_version'] = jwst_version
```
This is the change I proposed in https://github.com/spacetelescope/jwst/pull/8066#issuecomment-1802465009

Reply:
If lines 975-979 are adding the version numbers of all those packages (python, numpy, scipy, etc.) of which the jwst pkg is just one, then I'm fine with leaving it as "jwst", rather than changing to "calibration_software_version". That way the naming is consistent (i.e. all package names) and there's no harm in including all those pkg versions, rather than just jwst, because obviously the results in the source_catalog depend on all of them, especially `photutils`.